### PR TITLE
Improve logging in onos-config southbound

### PR DESCRIPTION
**Pull request by adibrastegarnia**
I think Insecure attribute confused people. Perhaps it is better to change it to something like skip_verify

### CODE REVIEW

**Review by frouzbeh — APPROVED** [review]

File: pkg/southbound/gnmi/conn.go [hunk 1]
```diff
@@ -69,14 +69,15 @@ func newDestination(target *topoapi.Object) (*baseClient.Destination, error) {
...
 	} else {
+		log.Infof("TLS connection to %s", configurable.Address)
 		tlsConfig := &tls.Config{}
 		if tlsOptions.Insecure {
-			log.Info("Insecure TLS connection to ", configurable.Address)
+			log.Infof("Skipping verification of certificate chain for target %s", configurable.Address)
 			tlsConfig = &tls.Config{InsecureSkipVerify: true}
 		} else {
-			log.Info("Secure TLS connection to ", configurable.Address)
+			log.Info("Enabled verification of certificate chain for target %s", configurable.Address)
```
Comment (SeanCondon) [inline pkg/southbound/gnmi/conn.go:80]: I think you should mention TLS somewhere in the message

File: pkg/southbound/gnmi/conn.go [hunk 2]
```diff
@@ -69,14 +69,15 @@ func newDestination(target *topoapi.Object) (*baseClient.Destination, error) {
...
 
 	if tlsOptions.Plain {
-		log.Info("Plain (non TLS) connection to ", configurable.Address)
+		log.Infof("Plain (non TLS) connection to %s", configurable.Address)
 	} else {
+		log.Infof("TLS connection to %s", configurable.Address)
 		tlsConfig := &tls.Config{}
 		if tlsOptions.Insecure {
-			log.Info("Insecure TLS connection to ", configurable.Address)
+			log.Infof("Skipping verification of certificate chain for target %s", configurable.Address)
```
Comment (SeanCondon) [inline pkg/southbound/gnmi/conn.go:77]: I think you should mention TLS somewhere in the message

File: pkg/southbound/gnmi/conn.go [hunk 3]
```diff
@@ -101,6 +102,7 @@ func newDestination(target *topoapi.Object) (*baseClient.Destination, error) {
 			tlsConfig.Certificates = []tls.Certificate{clientCerts}
 		} else if tlsOptions.Cert != "" && tlsOptions.Key != "" {
 			// Load certs given for device
+			log.Info("Loading the given certs for the device")
```
Comment (SeanCondon) [inline pkg/southbound/gnmi/conn.go:105]: also here

**Review by SeanCondon — APPROVED** [review]
minor comments

File: pkg/southbound/gnmi/conn.go (hunk 1 above)
Comment (adibrastegarnia) [reply pkg/southbound/gnmi/conn.go]: Done

File: pkg/southbound/gnmi/conn.go (hunk 2 above)
Comment (adibrastegarnia) [reply pkg/southbound/gnmi/conn.go]: Done

File: pkg/southbound/gnmi/conn.go (hunk 3 above)
Comment (adibrastegarnia) [reply pkg/southbound/gnmi/conn.go]: Done

**Pull request merged**